Versioning.md

```diff
@@ -1,32 +1,37 @@
 # Guidance on Coyote versioning
```

Comment:
Thanks for reverting the security file! And also for the "*" to "-" and "1)" to "1." (although they don't actually make any difference in the markdown output).

Reply:
I will approve and merge the PR once you revert the changes related to the few comments that I added.

Reply:
Should all be resolved now

Reply:
Thanks!